### ISSUE TITLE
/help/authors not up to date

### ISSUE THREAD

**Issue by adius** [purescript/pursuit#367]
```
Gzip the JSON output produced by psc-publish and save it to a file.
```

But `psc-publish` yields `Missing: --manifest FILE --resolutions FILE` and I can't find any documentation what those files are supposed to be

**Comment by adius** [comment]
According to the pulp code it seems to be:

```sh
bower list --json --offline > resolutions.json
psc-publish --manifest bower.json --resolutions resolutions.json
```

**Comment by adius** [comment]
Or maybe not. It has not the right format, but I can't really say any section in pulp where they change the format, so I don't really know...

**Comment by hdgarrood** [comment]
Yeah we should definitely document this. The idea was that it should be possible to produce a `resolutions.json` file without necessarily using bower. I think the right place for documenting this aspect of psc-publish would be the purescript/documentation repo.
